### ISSUE TITLE
Trouble creating movie folder with "Movie (Year) IMDBid"

### ISSUE THREAD

**Issue by mrnoisytiger** [Radarr/Radarr#3169]
I want Radarr to create my movie folders in the "Movie (Year) IMDBid" format for each movie I add. I already have all the existing movie folders in this format and I need the IMDBid for one of my automations. I have the naming convention set up in settings and allow it to automatically rename folders and create empty folders.

However, when I add movies manually Radarr seems to want to default the path to "Movie (Year)" only. When I add the movie, my logs are as follows:

![](https://i.imgur.com/l62CZah.png)

When I try to rescan the movie to get it to change the folder name, I get basically the same log entries.
For some reason, this isn't an issue with any movies I import via an IMDB list, but I also need it to work with Ombi and manually adding movies.

My settings are:

![](https://i.imgur.com/DMXLwY2.png)

**Comment by galli-leo** [comment]
See previous discussion on this issue.

TL;DR: Radarr doesn‘t know the imdbid at the time of adding the movie, hence it cannot create the path. The fix is not trivial and will require a rewrite which is being worked on.

**Comment by mrnoisytiger** [comment]
Thanks is this planned for 1.0? Could you also link me to the previous issue? I can't seem to find it. :(

**Comment by galli-leo** [comment]
https://github.com/Radarr/Radarr/projects/6

**Comment by galli-leo** [comment]
#153 Is a „catch-all“, but once that‘s „fixed“ your use case will work just fine.
